extensions/cornerstone/src/init.js

```diff
@@ -255,26 +287,31 @@ const _connectToolsToMeasurementService = measurementService => {
               measurement
             );
             console.log('Mapped annotation:', annotation);
+
```

Comment:
Include same check for event source type
This would _could_ be different than the measurement's `source.id`, as a non-source (measurement panel) could update the measurement.